src/pudl/metadata/fields.py

```diff
@@ -4176,6 +4176,50 @@
             "pattern": r"^\d{4}$",
         },
     },
+    "design_wind_speed_mph": {
+        "type": "number",
+        "description": "Design average annual wind speed for turbines at this generator.",
+    },
+    "obstacle_num_faa": {
+        "type": "string",
+        "description": "The FAA obstacle number assigned to this generator. If more than one obstacle number exists, the one that best represents the turbines. This field was only reported for a few years.",
+    },
+    "predominant_turbine_manufacturer": {
+        "type": "string",
+        "description": "Name of predominant manufacturer of turbines at this generator.",
+    },
+    "predominant_turbine_model": {
+        "type": "string",
+        "description": "Predominant model number of turbines at this generator.",
+    },
+    "turbine_hub_height_feet": {
+        "type": "number",
+        "description": "The hub height of turbines at this generator. If more than one value exists, the one that best represents the turbines.",
+        "unit": "ft",
+    },
+    "wind_quality_class": {
+        "type": "integer",
+        "description": "The wind quality calss for turbines at this generator.",
+        "constraints": {"enum": [1, 2, 3, 4]},
+    },
+    "wind_speed_avg_mps": {
+        "type": "number",
+        "description": "Average wind speed in miles per second.",
```

Comment:
No, there was no accompanying info about any of these fields from the table that described the wind class. I did poke around on the internet to try to find more information but it wasn't particularly conclusive.

Reply:
I think this has got to be meters per second, not miles per second. Otherwise these numbers would be hypersonic.

![image](https://github.com/catalyst-cooperative/pudl/assets/596279/260708b9-7734-4f19-a28c-2d331d134857)

Reply:
lol duh